js/utils.js

```diff
@@ -394,6 +394,36 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     }
   }
 
+  function moveAddbuttonPosition(instance) {
```

Comment:
Always use an options object with the utility functions because users can modify and customise layout JS and it would be a lot harder to extend the util functions later on if the customised layout JS don't assume an options parameter.

```suggestion
  function moveAddbuttonPosition(options) {
```

Reply:
@tonytlwu Fixed